echopype/tests/visualize/test_plot.py

```diff
@@ -212,9 +214,10 @@ def test_plot_mvbs(
         (30.5, False),
     ],
 )
-def test_water_level_echodata(water_level, expect_warning):
+def test_water_level_echodata(water_level, expect_warning, caplog):
```

Comment:
I don't understand what `caplog` is and where it's assigned :confused:  .  Could you explain, briefly? (Same goes for `test_water_level_Sv_dataset`).

I've run the test suite and all tests are passing, so I'm not concerned that something is failing. I just can't figure out what's going on.

Reply:
`caplog` is a special pytest attribute to get at captured logs by pytest test. See documentation: https://docs.pytest.org/en/7.1.x/how-to/logging.html#caplog-fixture

Reply:
Ah!! Thanks for the explanation.